### PR TITLE
fix/docs(Switch): fix uncontrolled examples

### LINKED ISSUE #7255: Bug - [Switch] - [uncontrolled examples are not working]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
The uncontrolled switch examples do not toggle.

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.
https://www.patternfly.org/v4/components/switch#uncontrolled

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #7255

@tlabaj Did these work before? Uncontrolled inputs shouldn't be setting `isChecked`, and half of the example switches are redundant now that it works. I'm curious if this was meant as a static demo initially.

### CODE REVIEW

File: packages/react-core/src/components/Switch/examples/Switch.md [hunk 1]
```diff
@@ -176,19 +176,9 @@ import { Switch } from '@patternfly/react-core';
     aria-label="Message when on"
     label="Message when on"
     labelOff="Message when off"
-    isChecked
+    isChecked={null}
```
Comment (jenny-s51) [inline packages/react-core/src/components/Switch/examples/Switch.md:179]:
```suggestion
    isChecked={null}
    defaultChecked
```

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Great update @kmcfaul !! It's really cool that we can get an uncontrolled input by setting isChecked to null! 

According to the react docs, it looks like we can [specify an initial value using defaultChecked but leave subsequent updates uncontrolled](https://reactjs.org/docs/uncontrolled-components.html#default-values).  

I see some `defaultChecked` logic in Checkbox.tsx`, `Radio.tsx`, both of which also have uncontrolled examples, maybe we can expose it in the props and use it in a similar way here? WDYT?

**Review by tlabaj — COMMENTED** [review]
@kmcfaul I am not sure if it did work before.  I would need tog look at the history

File: packages/react-core/src/components/Switch/examples/Switch.md [hunk 2]
```diff
@@ -176,19 +176,9 @@ import { Switch } from '@patternfly/react-core';
     aria-label="Message when on"
```
Comment (nicolethoen) [inline packages/react-core/src/components/Switch/examples/Switch.md:176]: Should we also update the aria-label when appropriate to demonstrate the intention? it's possible that, when it's uncontrolled, there are a few extra considerations to make to make sure the aria-label is meaningful. Like, what should we have the aria-label say if it wont change when the switch is on/off?

**Review by tlabaj — CHANGES_REQUESTED** [review]
Hmmm.  Katie, I think you were right, this probably never worked before.  I would expected it to work like the checkbox or radio in that you do not need to pass `isChecked` to make it uncontrolled.  I think we need to update the code so that is the case.  We can also support `isChecked` being passed as `null` in order to not break people...

**Comment by tlabaj** [comment]
@kmcfaul it looks like this issue came up before, there was an attempt to fix it that was reverted.  Here is the original issue https://github.com/patternfly/patternfly-react/issues/2791
See this related issue too
https://github.com/patternfly/patternfly-react/issues/2879

File: packages/react-core/src/components/Switch/examples/Switch.md (hunk 2 above)
Comment (thatblindgeye) [reply packages/react-core/src/components/Switch/examples/Switch.md:176]: I opened issue #7468 for this issue as well as an issue I noticed with the `aria-labelledby` attribute.

**Review by tlabaj — APPROVED** [review]
LGTM

**Review by jenny-s51 — APPROVED** [review]
LGTM!  👍  Nice work on this @kmcfaul @thatblindgeye

**Pull request merged**